Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/components/batteryInput/index.tsx

```diff
@@ -1,13 +1,28 @@
-const BatteryInput = () => {
+interface Props {
```

Comment:
You define the Props interface here, but don't use it in line 6 as the type of your props